apps/parsley/src/constants/logURLTemplates.ts

```diff
@@ -134,3 +134,10 @@ export {
   getEvergreenTestLogURL,
   getResmokeLogURL,
 };
+
+export const getEvergreenAllLogsURL = (
```

Comment:
If this is just for resmoke, could we update the name of this function? I think it's confusing because we also have "All logs" option for evergreen task logs